lib/config/browser-options.js

```diff
@@ -207,6 +208,18 @@ function buildBrowserOptions(defaultFactory, extra) {
             }
         }),
 
+        screenshotDelay: option({
+            defaultValue: defaultFactory('screenshotDelay'),
+            parseEnv: Number,
+            parseCli: Number,
+            validate: (value) => {
+                is('number')(value);
```

Comment:
you can use `assertNonNegative(value, 'screenshotDelay')` instead of those checks

Reply:
Done.